<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataView</class>
 <widget class="QWidget" name="DataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_dataView" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="themeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>主题</string>
        </property>
        <property name="text">
         <string>主题:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="themeComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="animatedLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>动画</string>
        </property>
        <property name="text">
         <string>动画:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="animatedComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="legendLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>图例Legend</string>
        </property>
        <property name="text">
         <string>图例:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="legendComboBox"/>
      </item>
      <item>
       <widget class="QCheckBox" name="antialiasCheckBox">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Anti-aliasing抗锯齿</string>
        </property>
        <property name="text">
         <string>抗锯齿</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_test">
        <property name="text">
         <string>test1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_chartView">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_view">
         <attribute name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/asset/dataview/MoreFilled.svg</normaloff>:/asset/dataview/MoreFilled.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>选项</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>91</width>
            <height>201</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_chart">
           <item>
            <widget class="QCheckBox" name="checkBox_current">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>电流选择是否显示</string>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>电流</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/asset/dataview/current.svg</normaloff>:/asset/dataview/current.svg</iconset>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_voltage">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>电流选择是否显示</string>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>电压</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/asset/dataview/voltage.svg</normaloff>:/asset/dataview/voltage.svg</iconset>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_pressure">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>电流选择是否显示</string>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>气压</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/asset/dataview/pressure.svg</normaloff>:/asset/dataview/pressure.svg</iconset>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_temperature">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>电流选择是否显示</string>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>温度</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/asset/dataview/Temperature.svg</normaloff>:/asset/dataview/Temperature.svg</iconset>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_setting">
         <attribute name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/asset/dataview/Tools.svg</normaloff>:/asset/dataview/Tools.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>设置</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_view">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>40</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>test</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
